quickstart-template/index.js

```diff
@@ -24,6 +24,4 @@ const transfer = await wallet.createTransfer({
 });
 
 // Wait for the transfer to complete or fail on-chain.
-await transfer.wait();
```

Comment:
@SeanMGonzalez we added `wait()` on transfer object in the 0.1.0 release.
docs - https://coinbase.github.io/coinbase-sdk-nodejs/classes/coinbase_transfer.Transfer.html#wait

 Could you run `npm install` from the quickstart-template directory and retry?